src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  hidden = false;
```

Comment:
The 'hidden' property should not be declared in the base 'Animal' class as it is specific to 'Herbivore' class. It should be moved to the 'Herbivore' constructor.

Reply:
Fix this comment